### PR TITLE
regression.py: Fix the error in running regression.py

### PR DESCRIPTION
**Pull request by quanwenli**
ID: 1866
Signed-off-by: Wenli Quan <wquan@redhat.com>

### CODE REVIEW

**Comment by quanwenli** [comment]
@PaulYuuu , could you help view, thanks

**Comment by PaulYuuu** [comment]
@quanwenli, can you explain more in the commit and fix different errors in different commits? Without more info, I don't know what this wants to do and how to review it.

**Comment by quanwenli** [comment]
> @quanwenli, can you explain more in the commit and fix different errors in different commits? Without more info, I don't know what this wants to do and how to review it.

I have updated, please help review. thanks.

File: scripts/regression.py [hunk 1]
```diff
@@ -134,10 +134,7 @@ def generate_raw_table(test_dict):
             partitions = getoutput("cat %s/partitions" % sysinfodir)
             fdisk = getoutput("cat %s/fdisk_-l" % sysinfodir)
 
-            status_path = os.path.join(os.path.dirname(files[0]), "../status")
-            status_file = open(status_path, 'r')
-            content = status_file.readlines()
-            self.testdata = re.findall("localtime=(.*)\t", content[-1])[-1]
+            self.testdata = arg.split("\n")
```
Comment (PaulYuuu) [inline scripts/regression.py:137]:
```suggestion
            self.testdata = arg.splitlines()
```

File: scripts/perf.conf [hunk 2]
```diff
@@ -8,8 +8,8 @@ avg_update =
 [netperf]
 result_file_pattern = .*.RHS
 ignore_col = 2
-avg_update = 4,2,3|12,5,10|13,6,11
-desc = The tests are *%s* seconds sessions of 'Netperf'. 'throughput' was taken from netperf's report.\nOther measurements were taken on the host.\nHow to read the results:\n - The Throughput is measured in Mbit/sec.\n - io_exit: io exits of KVM.\n - irq_inj: irq injections of KVM.\n
+avg_update =
```
Comment (PaulYuuu) [inline scripts/perf.conf:11]: It's there any reason to empty it?

**Review by PaulYuuu — COMMENTED** [review]
I can only give some basic review, as this is a standalone script, and the previous functional modification was in 2014, not sure how many people use it.

@YongxueHong, also requires your review.

File: scripts/regression.py (hunk 1 above)
Comment (quanwenli) [reply scripts/regression.py]: I have updated.

File: scripts/perf.conf (hunk 2 above)
Comment (quanwenli) [reply scripts/perf.conf]: I just update it to "avg_update = 4,2,3" since this is no other values in the column need to be calculated for the average.

**Comment by YongxueHong** [comment]
> I can only give some basic review, as this is a standalone script, and the previous functional modification was in 2014, not sure how many people use it.
> 
> @YongxueHong, also requires your review.

CC @luckyh

**Review by luckyh — APPROVED** [review]

**Pull request merged**